### PR TITLE
Fixes of deploy with Travis CI

### PR DESCRIPTION
**Pull request by rafaelje**
This PR work with:

- add package information
- improve travis scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@teclib/flyve-mdm-android-inventory-agent",
+  "version": "2.0.0",
+  "description": "Flyve Inventory Agent for Android",
```
Comment (ajsb85) [inline package.json]: Flyve MDM

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teclib/flyve-mdm-android-inventory-agent",
   "version": "2.0.0",
-  "description": "Flyve Inventory Agent for Android",
+  "description": "Flyve MDM Inventory Agent for Android",
```
Comment (ajsb85) [inline package.json]: It's not a "refactor" commit type, you can use "ci", because the package JSON is used to generate the version number and changelog.

Comment (ajsb85) [reply package.json]: Refactor is used for the Java source code in this case.

**Comment by nigon135** [comment]
on pull request

File: package.json (hunk 2 above)
Comment (nigon135) [reply package.json]: Refactor is used for the Java source code in this case.

File: package.json (hunk 1 above)
Comment (nigon135) [reply package.json]: Flyve MDM

**Pull request merged**